sssd_test_framework/utils/pam.py

```diff
@@ -0,0 +1,235 @@
+""""PAM Tools."""
+
+from __future__ import annotations
+
+from pytest_mh import MultihostHost, MultihostUtility
+from pytest_mh.utils.fs import LinuxFileSystem
+
+__all__ = [
+    "PAMUtils",
+    "PAMAccess",
+    "PAMFaillock",
+]
+
+
+class PAMUtils(MultihostUtility[MultihostHost]):
+    """
+    Management of PAM modules
+    """
+
+    def __init__(self, host: MultihostHost, fs: LinuxFileSystem) -> None:
+        """
+        :param host: Remote host instance
+        :type host: MultihostHost
+        """
+        super().__init__(host)
+
+        self.file = None
+        self.fs: LinuxFileSystem = fs
+
+    def access(self, file: str | None = None) -> PAMAccess:
```

Comment:
Don't use None but `file: str = "/etc/security/access.conf"`. Then you can just use `return PAMAccess(self, file)`.